### PR TITLE
OSSSIO-CIDM-NEO-profile-1.0.1_20170728T080953-DEV

### PR DESCRIPTION
**Pull request by eddo888**
this is the original wrapped widget (without the persistent cross site hub)
releasing a named version for backward compatibility
https://servicesmadesimpler.govnet.qld.gov.au/jira/browse/QGCIDM-569

### CODE REVIEW

**Review by naveen2803 — COMMENTED** [review]
1. Can we make the URL consistent for FAQ, ABOUT and MYACCOUNT in OSSSIO/CIDM-NEO/profile/1.0.1_20170728T080953/DEV/index.html

2. It will be good if we reference lock.min.js rather than lock.js in minimal.html as it will speed up loading time.

File: OSSSIO/CIDM-NEO/profile/1.0.1_20170728T080953/DEV/callback.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  <head>
+    <title>My Agency</title>
+  </head>
+  <body>
+
+    <script type="text/javascript">
+
+var hash = window.location.href.split('#')[1];
+var callback = localStorage.getItem('callback');
+if (callback != null) {
```
Comment (gvanimurthy) [inline OSSSIO/CIDM-NEO/profile/1.0.1_20170728T080953/DEV/callback.html:11]: Could we check if hash is undefined as well? If has is undefined callback url would be callback.html#undefined.
```if (callback != null ) {
		if( hash !== undefined) {
			window.location.replace(callback + '#' + hash);
		}
		else {
			window.location.replace(callback);
		}
	}```

File: OSSSIO/CIDM-NEO/profile/1.0.1_20170728T080953/DEV/hub.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+<head>
+  <title>Cross Storage Hub</title>
+</head>
+<body>
+  <script type="text/javascript" src="http://localhost:8080/js/csls/hub.js"></script>
```
Comment (gvanimurthy) [inline OSSSIO/CIDM-NEO/profile/1.0.1_20170728T080953/DEV/hub.html:6]: Could this be hub.min.js?

File: OSSSIO/CIDM-NEO/profile/1.0.1_20170728T080953/DEV/index.html [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <link rel="icon" type="image/png" href="qld-crest-on-top-2linestacked-b-w.png" />
+
+    <!-- cidm-neo.css introduces style changes to allow the avatar to display as a horizontal list -->
+    <link rel="stylesheet" type="text/css" href="https://beta-static.qgov.net.au/assets/widget/CIDM/profile/latest/css/cidm-neo.css"></link>
+    <link rel="stylesheet" type="text/css" href="https://beta-static.qgov.net.au/assets/widget/CIDM/profile/latest/css/cidm-profile.css"></link>
+    <link rel="stylesheet" type="text/css" href="https://beta-static.qgov.net.au/assets/widget/CIDM/profile/latest/css/swe3-override.css"></link>
+    <link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
```
Comment (gvanimurthy) [inline OSSSIO/CIDM-NEO/profile/1.0.1_20170728T080953/DEV/index.html:15]: Use min.css http://code.jquery.com/ui/1.12.1/themes/base/jquery-ui.min.css

File: OSSSIO/CIDM-NEO/profile/1.0.1_20170728T080953/DEV/index.html [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+
+<head>
+    <title>CIDM Wrapped Widget Resources</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <link rel="icon" type="image/png" href="qld-crest-on-top-2linestacked-b-w.png" />
+
+    <!-- cidm-neo.css introduces style changes to allow the avatar to display as a horizontal list -->
+    <link rel="stylesheet" type="text/css" href="https://beta-static.qgov.net.au/assets/widget/CIDM/profile/latest/css/cidm-neo.css"></link>
```
Comment (gvanimurthy) [inline OSSSIO/CIDM-NEO/profile/1.0.1_20170728T080953/DEV/index.html:12]: Could these 3 minified and combined to make a single server request?

**Review by naveen2803 — APPROVED** [review]

**Pull request merged**